napari/_qt/_qapp_model/_tests/test_qdispatcher.py

```diff
@@ -0,0 +1,140 @@
+import time
+from unittest.mock import MagicMock, patch
+
+from app_model.registries import CommandsRegistry
+
+from napari._qt._qapp_model._qdispatcher import (
+    PRESS_HOLD_DELAY_MS,
+    QKeyBindingDispatcher,
+)
+from napari.utils.key_bindings import DispatchFlags
+
+
+def test_q_key_binding_dispatcher(qtbot):
```

Comment:
Is it required to have so long test function? It looks like it could be simply split on multiple test functions.